tests/integration/test_iam.py

```diff
@@ -356,3 +352,21 @@ def test_create_role_with_assume_role_policy(self, iam_client):
         assert roles["Roles"][0]["Path"] == "myPath"
         assert roles["Roles"][0]["RoleName"] == role_name_2
         assert len(roles["Roles"]) == 1
+
+    @pytest.mark.aws_validated
+    @pytest.mark.xfail
+    @pytest.mark.parametrize(
+        "service_name, expected_role",
+        [
+            ("ecs.amazonaws.com", "AWSServiceRoleForECS"),
+            ("ecr.amazonaws.com", "AWSServiceRoleForECR"),
+        ],
```

Comment:
would be nice if we could extend that at some point in the future to cover all services :thinking: